.github/workflows/coverage.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           git push
         fi
     - name: Upload Jacoco coverage report
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v4
```

Comment:
Could you run `git pull upstream --rebase main` so that this change from a previous PR does not appear in this PR? Thanks.

Reply:
I'll keep in mind before pushing origin! Thanks